### DISCUSSION TITLE
Running quartznet encoder in production

### DISCUSSION THREAD

**Discussion by abhinavdayal** [NVIDIA-NeMo/NeMo#4014] [Q&A]
Hi,

I am working on a specific problem, where I am more interested in the encoder part and want to access the emission probabilities than do a CTC decoder to get the final transcription.

```
logits = asr_model.transcribe(files, logprobs=True)[0]
```
I have a post process to analyze the logits and produce output that is specific to the application. 

If I have to run this in production code, what will be appropriate? Is there any example of something like that? Currently I am running nemo asr directly.

**Comment by titu1994** [comment]
Nemo models are not used for production but depends on your definition of production. If it works and gets the results you need then you can use it as such. For proper production deployment, please look into Nvidia RIVA.
